aten/src/ATen/native/Convolution.cpp

```diff
@@ -1434,6 +1434,11 @@ static inline at::MemoryFormat determine_backend_memory_format(
         backend_memory_format = at::MemoryFormat::ChannelsLast;
       }
       break;
+    case ConvBackend::Overrideable:
+      if (xpu_conv_use_channels_last(input, weight)) {
+        backend_memory_format = at::MemoryFormat::ChannelsLast;
+      }
```

Comment:
The logic should not impact other backends as the `Overrideable` is a general backend for other out-of-tree backends. Please set the `backend_memory_format` to `Contiguous` if the input and weight are not `XPU.`

Reply:
`backend_memory_format` is `Contiguous` by default.

Reply:
Besides, please check the dim number just like what mkldnn backend has done. If the k is 5, the mem format should be channels-last 3d.

Reply:
done.